frontend/src/components/KBar/KBarWrapper.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <KBarProvider :actions="globalActions" :options="{ disabled }">
+    <KBarPortal>
+      <KBarPositioner class="mask">
+        <KBarAnimator class="container">
+          <KBarSearch class="searchbox" />
+          <KBarHelper />
+          <RenderResults />
+          <KBarFooter />
+        </KBarAnimator>
+      </KBarPositioner>
+    </KBarPortal>
+
+    <slot />
+  </KBarProvider>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed } from "vue";
+import {
+  KBarProvider,
+  KBarPortal,
+  KBarPositioner,
+  KBarAnimator,
+  KBarSearch,
+  createAction,
+} from "@bytebase/vue-kbar";
+import { useStore } from "vuex";
+import { useRouter } from "vue-router";
+import RenderResults from "./RenderResults.vue";
+import KBarHelper from "./KBarHelper.vue";
+import KBarFooter from "./KBarFooter.vue";
+
+export default defineComponent({
+  name: "KBarWrapper",
+  components: {
+    KBarProvider,
+    KBarPortal,
+    KBarPositioner,
+    KBarAnimator,
+    KBarSearch,
+    RenderResults,
+    KBarHelper,
+    KBarFooter,
+  },
+  setup() {
+    const store = useStore();
+    const router = useRouter();
+
+    const disabled = computed(() => {
+      const currentUser = store.getters["auth/currentUser"]();
+      // totally disable kbar when not logged in
```

Comment:
Disable kbar when not logged in since there is no point to show it on login/signup page.

Reply:
Yes, already checked. The condition is `!currentUser || currentUser.id < 0`.

Reply:
yeah, I mean to change the comment to be more explicit.

Reply:
updated